examples/evmapp/src/main/resources/sc_evm_settings.conf

```diff
@@ -50,7 +50,7 @@ scorex {
 
     wallet {
         seed = "sidechain_seed_0"
-        genesisSecrets = ["031ec02e4e14c4597f60948ddba044e3be42af6f1f613e8b926013d7da5d617c1fb9aa4434d312ba0a232c267c9df46448b10571d229882e3feb0eab5491a4690380", "003868b579a763b620cabe5882dae3c39be8c84719d6c2ba4ad2ab4b0653aacd04a5b10622d70f094b7276e04608d97c7c699c8700164f78e16fe5e8082f4bb2ac", "0537fe08d6d4a845973222db81c64b6479dab45e1a974029bdbeee6be6db553d63"]
+        genesisSecrets = ["032aa8faf5697ea9f68cd1bf9b5958c8dff646d30811fba11d19511a795352fa11f666c94e29942c19d31b8bfba9c34d6d8a3468418b6606cc6896b40bbe5dc70400", "003868b579a763b620cabe5882dae3c39be8c84719d6c2ba4ad2ab4b0653aacd04a5b10622d70f094b7276e04608d97c7c699c8700164f78e16fe5e8082f4bb2ac", "054da10f377a9bb734f8313b4074cd0902710501f1fbdf29e8c1d76f17dee716bb"]
```

Comment:
We also need to update the data of `withdrawalEpochCertificate` group. Otherwise if you will start EvmApp with this config you will see the following error: `com.horizen.account.certificatesubmitter.AccountCertificateSubmitter - Backward transfer certificate submitter failed to start due: ...`